app/views/items/_item.html.erb

```diff
@@ -1 +1,3 @@
-<li id="item-<%= item.id %>"><%= item.name %><%= link_to "", user_item_path(current_user, item), method: :delete, remote: true, class: 'glyphicon glyphicon-ok' %></li>
+<tr>
+  <td><li id="item-<%= item.id %>"><%= item.name %>&nbsp<%= link_to "", user_item_path(current_user, item), method: :delete, remote: true, class: 'glyphicon glyphicon-ok' %>&nbsp<%= distance_of_time_in_words(item.created_at, item.created_at + 168.hours) %></li></td>
```

Comment:
You probably should have put each of these in their own table cells rather than separating with a space.